### PR TITLE
[plugin.video.nhlgcl] 2017.1.24

### PR DESCRIPTION
**Pull request by eracknaphobia**
### Description

2016.1.24
- Updated addon icon and fanart images
- Updated favorite team image
- Added player mugshot to goal notifications
- Fixed Cache issue with adding favorite team link to the Kodi home screen
- Fixed addon.xml forum link
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.video.nhlgcl/addon.xml [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
         <provides>video</provides>
   </extension>  
   <extension point="xbmc.addon.metadata">    
     <platform>all</platform>
-    <summary lang="en">Watch out-of-market regular season games of every team LIVE from around the league</summary>
-    <description lang="en">Watch out-of-market regular season games of every team LIVE from around the league</description>
+    <summary lang="en">Watch out-of-market games from around the League on Kodi, with HD quality streaming</summary>
+    <description lang="en">With NHL.TV you can access revolutionary 60fps (frames per second) video through Kodi</description>
     <disclaimer lang="en"></disclaimer>
     <language>en</language>
```
Comment (enen92) [inline plugin.video.nhlgcl/addon.xml]: JFYI you can also use the news tag when you update a plugin (in addition to the changelog)

File: plugin.video.nhlgcl/addon.xml [hunk 2]
```diff
@@ -1,23 +1,24 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.nhlgcl" name="NHL TV™" version="2016.11.18" provider-name="eracknaphobia">
+<addon id="plugin.video.nhlgcl" name="NHL TV™" version="2017.1.24" provider-name="eracknaphobia">
   <requires>
    <import addon="xbmc.python" version="2.19.0"/>
```
Comment (enen92) [inline plugin.video.nhlgcl/addon.xml]: When you have some time (next update) please consider setting the xbmc.python version to 2.24.0  so it matches jarvis

**Comment by enen92** [comment]
Thanks

File: plugin.video.nhlgcl/addon.xml (hunk 1 above)
Comment (eracknaphobia) [inline plugin.video.nhlgcl/addon.xml]: Ok, I didn't know such a tag existed. I will take a look.

File: plugin.video.nhlgcl/addon.xml (hunk 2 above)
Comment (eracknaphobia) [inline plugin.video.nhlgcl/addon.xml]: Sure thing. I've changed it and it will be in the next release.

**Pull request merged**